### DISCUSSION TITLE
I'm getting WEBUSB is not supported error when trying go to from a different machine on the network to my host machine.

### DISCUSSION THREAD

**Discussion by sbaccam75** [yume-chan/ya-webadb#577] [General]
Hello I was wondering if you know why this error would occur. I am using the newest chrome version 115.0.5790.114.

**Comment by yume-chan** [comment]
See https://github.com/yume-chan/ya-webadb/blob/main/CONTRIBUTING.md#1-webusb-and-file-downloading-doesnt-work-in-developmentself-host-environment
